.bazelrc

```diff
@@ -238,7 +238,7 @@ common:macos --workspace_status_command=$(pwd)/workspace_status.sh
 common:windows --workspace_status_command="bash workspace_status.sh"
 
 # Misc remote cache/BES optimizations
-common --experimental_remote_cache_async
+common --remote_cache_async
```

Comment:
question: is this flag gona be a problem in CI where we shutdown container/vm as Bazel CLI exit?

Reply:
No, it just makes it so that uploads don't block action execution, but downloads will still complete before the CLI exits.